handler.go

```diff
@@ -89,6 +89,14 @@ type StickerMessageHandler interface {
 	HandleStickerMessage(message StickerMessage)
 }
 
+/*
+The ContactMessageHandler interface needs to be implemented to receive location messages dispatched by the dispatcher.
```

Comment:
are this Location Messages correct?

Reply:
*on comments

Reply:
Typo 😬
I'll fix asap

Reply:
I created another pull request adding the proto messages to be able to send ContactMessages.

You all can decide to merge or not... in the next time I`ll sen da pr to your branch instead of create a new merge .. sorry about that